tests/5.0/target/test_target_defaultmap_none.F90

```diff
@@ -0,0 +1,95 @@
+!===--- test_target_defaultmap_none.F90 -------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+!   This test checks behavior of the defaultmap clause when the specified implicit-behavior  
+!   is none. The variable-categories avaiable for defaultmap are scalar, aggregate, allocatable,
+!   and pointer. When defaultmap(none) is used alone without a variable-category, then none is 
+!   the implicit-behavior for all variables referenced in the construct. Since none is used, 
+!   the data mapping attributes must be specified somewhere else. In this case, they are 
+!   specified using the map clause.
+!
+!//===----------------------------------------------------------------------===//
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_target_defaultmap_none
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_TEST_VERBOSE(defaultmap_none() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION defaultmap_none()
+    TYPE structure
+       INTEGER :: s
+       INTEGER,DIMENSION(N) :: SA
+    END TYPE structure
+
+    INTEGER :: errors, i
+    INTEGER :: scalar !scalar
+    INTEGER, TARGET, DIMENSION(N) :: A !aggregate
+    INTEGER, ALLOCATABLE :: B(:) !allocatable
+    INTEGER, POINTER :: ptr(:) !pointer
+
+
+    TYPE(structure) :: new_struct !aggregate
+    ALLOCATE(B(N))
+
+    scalar = 1
+    new_struct%s = 1
+
+    DO i = 1, N
+       A(i) = 0
+       B(i) = 0
+       new_struct%SA(i) = 0
+    END DO
+
+    errors = 0
+
+    !$omp target defaultmap(none) map(tofrom: scalar, A, B, new_struct, ptr)
+    scalar = 17 !scalar firstprivate, value not returned
```

Comment:
I think if to from is explicit then the scalar value will be copied back.

Reply:
The comment seems to be wrong; what I wanted to say was that scalar variables without TARGET, ALLOCATABLE, or POINTER attribute have an implicit data-sharing attribute of first-private. (OpenMP V5.0 page 315 line 9 ~ 12). Is it correct interpretation?

Reply:
@seyonglee IMO that is the correct interpretation, I see what you were implying with your comment. @spophale This looks good-to-go. I will approve and leave it up for you to merge. Thanks again @seyonglee !

Reply:
Changed the comments for the scalar variable to clarify the intention.